paper-server/patches/sources/net/minecraft/server/MinecraftServer.java.patch

```diff
@@ -779,6 +779,15 @@
 +                LOGGER.info("*************************************************************************************");
 +            }
 +            // Paper end - Add onboarding message for initial server start
++            // Paper start - improve outdated version check
++            if (System.getProperty("IReallyKnowWhatIAmDoingISwear") == null) {
```

Comment:
the system proper changed its meaning and thus should be named better now, something like paper.disableStartupVersionCheck

Reply:
also done